### ISSUE TITLE
Bug: Contribution page shows incorrect PR label.

### ISSUE THREAD

**Issue by Mandeep56Singh** [open-sauced/app#4057]
### Describe the bug

I made a PR which was initially "Draft" but after some time I changed it to "Open"(ready to review). But it seems data is not updated. It still shows draft label.

https://github.com/user-attachments/assets/ddac9b15-74b6-47a7-96ed-14a98756e65c

### Steps to reproduce

1. Go to [https://app.opensauced.pizza/u/Mandeep56Singh](https://app.opensauced.pizza/u/Mandeep56Singh)
2. see under contributions, label for PR, its draft
3. click on the PR
4. PR is Open.

**Comment by nickytonline** [comment]
@Mandeep56Singh, thanks for the PR, but please read our contributing guidelines. Issues are required for a PR and you need to be assigned the issue before opening up a PR related to the issue. 

Also, it will update the status of the PR on your profile page. It's not instantaneous.

**Comment by Mandeep56Singh** [comment]
Hi @nickytonline, thank you for your response!

Just to clarify, I was indeed assigned to the issue, and I followed the contributing guidelines. The concern I’m raising is about the status update of the PR on the contribution page.

I’ve noticed that even after changing the PR from "Draft" to "Open," it still shows as "Draft" on the contribution page, even after 6 days. I made a short video that demonstrates the issue, showing both the current PR that was initially a draft and then made open, as well as the one from 6 days ago.


https://github.com/user-attachments/assets/09d62429-b2aa-47c8-b220-410c5869917f





Could this be a bug, or is there something I might be missing? Any advice or assistance would be greatly appreciated!

Thanks again!
